### ISSUE TITLE
UploadSetting CacheControl is not setting

### ISSUE THREAD

**Issue by Jeff-Stapleton** [SharpeRAD/Cake.AWS.S3#20]
I am trying the set CacheControl to "no-cache" but no matter what I do CacheControl always reverts to "public, max-age=864000"

```        
var settings = CreateSyncSettings(awsAccessKey, awsSecretKey)
            .SetBucketName(s3Bucket)
            .SetRegion(regionName)
            .SetLowerPaths(false);
        
settings.CacheControl = "no-cache";

var invalidationList = await S3SyncUpload((DirectoryPath)"site", settings);
```

**Comment by SharpeRAD** [comment]
Hi Jeff, can you check which version of Cake.AWS.S3 your using as I fixed this in **v0.6.5**. S3SyncUpload is now setting a custom Cache-Control header for our CDN 😄 

Please close the issue if upgrading resolves this for you.

**Comment by Jeff-Stapleton** [comment]
That did the trick! thanks.
